.storybook/main.js

```diff
@@ -0,0 +1,32 @@
+const config = {
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+
+  addons: ['@storybook/preset-create-react-app', '@storybook/addon-docs', '@storybook/addon-onboarding'],
+
+  framework: {
+    name: '@storybook/react-webpack5',
+    options: {},
+  },
+
+  docs: {
+    autodocs: true,
+  },
+
+  staticDirs: ['../public'],
+
+  webpackFinal: async (config) => {
+    // 🚫 Remove any ESLint loaders injected by CRA preset
+    config.module.rules = config.module.rules.filter(
+      (rule) => !(rule.use && rule.use.some((u) => u.loader?.includes('eslint')))
+    );
+
+    // 🚫 Remove ESLint plugin created by CRA (this is the real problem)
```

Comment:
Could you remove the emoji from the comments? We want to keep the comments sober, and also emojis might not be correctly visible in all IDEs 😄